File: help/eng_gb/exercicios_da_ficha_9/Ficha09_014/Ficha09_014.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Decimal : " );// print text
    num = parseInt(prompt("Decimal : " ,"0") );
    
    document.write(num + "\n");// print input 
    var binary = "" ;
    while(num > 0 ){
        binary = num % 2 + binary ;
        num /=2;
    }
    document.write("Binary = " + binary );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_014/Ficha09_014.py
# Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0


num =  int(input("Decimal : "))
binary = "" 
while num > 0  :
    binary =  str(num % 2) + binary
    num = num // 2
print( "Binary = ", binary  , end="")


#Python 3  - Automatic translated by Algorithmi 22.05
#(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12
